index.html

```diff
@@ -225,8 +225,9 @@ <h2>Usage</h2>
               [--flush-session] [--store-session PATH] [-s URL] [-x URL]
               [-r PARAMETER] [--skip PARAMETER] [-d DEPTH]
               [--max-links-per-page MAX] [--max-files-per-dir MAX]
-              [--max-scan-time MINUTES] [--max-parameters MAX] [-S FORCE]
-              [-t SECONDS] [-H HEADER] [-A AGENT] [--verify-ssl {0,1}]
+              [--max-scan-time SECONDS] [--max-attack-time SECONDS]
```

Comment:
You  should edit the wapiti.ronn file instead, this is essentially a markdown file and the manpage and its html version are generated from the ronn file.

Take a look at https://github.com/rtomayko/ronn

Reply:
sorry, did not saw it was the website repo ;-)